### PR TITLE
Self Service Reports : Added a switch to enable metabase setup , a button to check sync status and a button to refresh tables

### PR DESCRIPTION
**Pull request by ombhardwajj**

### CODE REVIEW

File: src/reports/SelfServiceReports.js [hunk 1]
```diff
@@ -11,81 +10,191 @@ import { makeStyles } from "@material-ui/core";
...
+  const [setupEnabled, setSetupEnabled] = useState(false);
+  const [loadingSetup, setLoadingSetup] = useState(false);
+  const [loadingRefresh, setLoadingRefresh] = useState(false);
+  const [syncStatus, setSyncStatus] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  useEffect(() => {
+    if (errorMessage) {
+      const timer = setTimeout(() => {
+        setErrorMessage("");
```
Comment (1t5j0y) [inline src/reports/SelfServiceReports.js:56]: No need to hide error message after 10 seconds. Use a toast if you want to do this.

**Review by 1t5j0y — COMMENTED** [review]
Minor feedback around clearing error message

**Comment by ombhardwajj** [comment]
Alright will try to use toast !

File: src/reports/SelfServiceReports.js (hunk 1 above)
Comment (ombhardwajj) [reply src/reports/SelfServiceReports.js:56]: Alright will try to use toast !

**Pull request merged**